subprojects/docs/src/docs/userguide/publishingIvy.adoc

```diff
@@ -79,16 +79,16 @@ The simplest way to publish a Gradle project to an Ivy repository is to specify
 
 | `java`
 | <<java_plugin,Java Plugin>>
-| Generated jar file
+| Generated JAR file
 | Dependencies from 'runtime' configuration
 
 | `web`
 | <<war_plugin,War Plugin>>
-| Generated war file
+| Generated WAR file
 | No dependencies
 |===
```

Comment:
Any chance of updating this table to a definition list? If so, please see my [pull request](https://github.com/gradle/gradle/pull/5263) for the Distribution Plugin chapter, which hopefully clarifies the current styling.